workflow-dockstore-metrics/workflow_dockstore_metrics/pull_and_send_metrics.py

```diff
@@ -0,0 +1,202 @@
+"""
+This script pulls workflow execution data from BigQuery and sends it to Dockstore.
+It then updates the metadata table to indicate that the data has been sent to Dockstore.
+
+Requires:
+ - signing into a firecloud account with access to the broad-dsde-prod-analytics-dev project
+ - activating the associated poetry env with `poetry shell`, then poetry install
+
+Args:
+--dry_run: Run in test mode. Does not send metrics to dockstore or update the metadata table.
+--login: default Re-authenticate with gcloud
+--lookback_window: How many days of metrics to send (default 2)
```

Comment:
When this is run using github actions will it still just be 2 day look back? Would we want to give larger default in case of failures, outages, etc.?

Reply:
In the GHA where we call this script we can specify the lookback window to whatever we think is best. The tradeoff is that the longer window we set, the longer and more expensive the query will be, but anything ~ 1 week or less should be fine. I can also make it an argument for the GHA so that in the case of an outage or something we can give a custom window